components/ws-proxy/pkg/sshproxy/server.go

```diff
@@ -43,18 +43,20 @@ var (
 )
 
 var (
```

Comment:
it would be nice to add comment to each code to specify meaning, i.e. auth with reject means that workspace is not running?